server/lib/job-queue/handlers/video-import.ts

```diff
@@ -144,6 +151,29 @@ async function processFile (downloader: () => Promise<string>, videoImport: MVid
     }
     videoFile = new VideoFileModel(videoFileData)
 
+    // Check we accept this video
+    const acceptParameters = {
+      videoFilePath: tempVideoPath,
+      videoFile,
+      user: videoImport.User
+    }
+    const acceptedResult = options.type === 'youtube-dl'
```

Comment:
Instead of using a ternary on the `wrapFun` call:

`let hookName = ... ? 'filter:api.video.post-import-url.accept.result': '...'`

And then just call the function `wrapFun(isPostImportVideoAccepted, acceptParameters, hookName)`